server/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const port = 3001;
```

Comment:
Would be good to set this to a conditional like:
`const port = process.env.PORT || 3001`
That way we won't need to change too much depending on the environment we are running in later on.

Reply:
Thanks for the feedback! I'll get that done.